### PR TITLE
SOV-2083: add D2 links

### PR DESCRIPTION
**Pull request by creed-victor**
https://sovryn.atlassian.net/browse/SOV-2083

### CODE REVIEW

File: src/assets/apps-icon.svg
Comment (pietro-maximoff) [inline src/assets/apps-icon.svg:1]: maybe it makes sense to make the icon a bit smaller, for instance, 32px?

File: src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx [hunk 1]
```diff
@@ -188,6 +191,33 @@ export const DefaultHeaderComponent: React.FC = () => {
       dataActionId: 'header-mobile-bitocracy-link-forum',
       hrefExternal: true,
     },
+    {
+      to: '',
+      title: 'sovryn.app',
```
Comment (pietro-maximoff) [inline src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx:196]:
```suggestion
      title: 'Sovryn.app',
```

**Review by pietro-maximoff — CHANGES_REQUESTED** [review]

File: src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx [hunk 2]
```diff
@@ -223,13 +253,16 @@ export const DefaultHeaderComponent: React.FC = () => {
         href={link.to}
         text={link.title}
         onClick={() => {
-          history.push(link.to);
           setOpen(false);
+          if (link.hrefExternal) return;
+          history.push(link.to);
```
Comment (tiltom) [inline src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx:258]:
```suggestion
          if (link.hrefExternal) {
            return;
          }

          history.push(link.to);
```

File: src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx [hunk 3]
```diff
@@ -308,6 +342,37 @@ export const DefaultHeaderComponent: React.FC = () => {
               </Link>
             </div>
             <Menu className="tw-hidden xl:tw-flex tw-flex-row tw-flex-nowrap 2xl:tw-space-x-4 tw-items-center">
+              <NavPopover
+                content={
+                  <>
+                    <MenuItem
+                      text={t(translations.mainMenu.borrow)}
+                      label={t(translations.mainMenu.labels.d2Borrow)}
+                      href="https://sovryn.app"
```
Comment (tiltom) [inline src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx:351]: Don't forget to change the links here as well.

**Review by tiltom — CHANGES_REQUESTED** [review]

**Review by tiltom — DISMISSED** [review]

**Review by pietro-maximoff — DISMISSED** [review]

**Pull request merged**